src/index.js

```diff
@@ -68,64 +67,58 @@ class Wurd {
    * Loads a section of content so that it's items are ready to be accessed with #get(id)
    *
    * @param {String|Array} ids      IDs of sections to load content for. Can be an array or comma-separated string of sections to load, e.g. 'main,home'
-   * @param {Object} [options]      Options to override the instance defaults.
+   * @param {Object} [opts]      Options to override the instance defaults.
    * 
    * @return {Promise}
    */
-  load(ids, options = {}) {
+  load(_ids, _options) {
     //Merge default and request options
-    options = Object.assign({}, this.options, options);
+    const options = { ...this.options, ..._options };
 
     //Force draft to true if in editMode
     if (options.editMode === true) {
       options.draft = true;
     }
 
-    return new Promise((resolve, reject) => {
-
-      let {app} = this;
+    const { app } = this;
 
-      if (!app) return reject(new Error('Use wurd.connect(appName) before wurd.load()'));
+    if (!app) return Promise.reject(new Error('Use wurd.connect(appName) before wurd.load()'));
```

Comment:
we could throw, if we declare load method as async `async load(_ids, _options) {`

Reply:
well let's do it anyway